modules/contact/src/userobjects/WeightedGapUserObject.C

```diff
@@ -126,8 +126,10 @@ WeightedGapUserObject::computeQpIProperties()
   // Get the _dof_to_weighted_gap map
   const auto * const dof = static_cast<const DofObject *>(_lower_secondary_elem->node_ptr(_i));
 
-  _dof_to_weighted_gap[dof].first += (*_test)[_i][_qp] * _qp_gap_nodal * _normals[_i];
-  _dof_to_weighted_gap[dof].second += (*_test)[_i][_qp] * _qp_factor;
+  auto & [weighted_gap, normalization] = _dof_to_weighted_gap[dof];
+
+  weighted_gap += (*_test)[_i][_qp] * _qp_gap_nodal * _normals[_i];
+  normalization += (*_test)[_i][_qp] * _qp_factor;
```

Comment:
I'm assuming this shouldn't be in this PR?

Reply:
Can't I sneak in a rider commit especially with someone familiar with contact? 😄

Reply:
Yup, this makes it so much more readable anyway!